SharedProjects/BabylonExport.Entities/ExportParameters.cs

```diff
@@ -15,6 +15,7 @@ public class ExportParameters
         public float scaleFactor = 1.0f;
         public bool writeTextures = true;
         public bool overwriteTextures = true;
+        public bool flattenExport = true;
```

Comment:
can we change this to "flattenScene"? It just gives a better idea of what it's used for. also try to homogenize usage of collapse and flatten to use flatten.

Reply:
done as well